src/reporters/udp_sender.js

```diff
@@ -148,7 +148,7 @@ export default class UDPSender {
     });
     this._reset();
```

Comment:
I think reset needs to be called at L134, after the spans are converted to thrift buffer

Reply:
Would this then close #214?